
string name
float64[3] position
string type
bool graspable
bool pushable
bool free
float64 size
string above_str
string under_str
# drawer stuff
bool opened
string contains_list
# cup
bool full
